pkg/meta/openfile.go

```diff
@@ -199,6 +199,10 @@ func (o *openfiles) ReadChunk(ino Ino, indx uint32) ([]Slice, bool) {
 	if !ok {
 		return nil, false
 	}
+	if time.Since(time.Unix(of.lastCheck, 0)) >= o.expire {
```

Comment:
The chunk cache should also work when `open-cache=0`.

Reply:
Oh, I thought this line has a bug, not realizing it's a feature...
https://github.com/juicedata/juicefs/blob/ad6cf901579d65d89075619f1116069dabe2bdd3/pkg/meta/base.go#L1459

But if we cache chunks even when `open-cache` is disabled, I suppose we are breaking the close-to-open semantics? The old cache remains even when user reopens the file - he gets a fresh attr but stale data?

Reply:
No, the chunk cache is invalidated in `Update`

Reply:
Got it. It's updated according to mtime.